pyproject.toml

```diff
@@ -1,17 +1,17 @@
 [tool.poetry]
 name = "onnx2kerastl"
-version = "0.0.86"
+version = "0.0.88.dev1"
 description = ""
 authors = ["dorhar <doron.harnoy@tensorleap.ai>"]
 license = "MIT"
 
 [tool.poetry.dependencies]
 python = ">=3.8, <3.11"
-tensorflow = {version = "2.11.0", markers = "platform_machine  == 'x86_64'"}
-tensorflow-macos = {version = "2.11.0", markers = "platform_machine  == 'arm64'"}
+tensorflow = {version = "2.12.0", markers = "platform_machine  == 'x86_64'"}
+tensorflow-macos = {version = "2.12.0", markers = "platform_machine  == 'arm64'"}
 onnx = "1.10.1"
 protobuf = "^3.19.6"
-keras-data-format-converter = "0.1.14"
+keras-data-format-converter = "0.1.15.dev3"
 tensorflow-addons = {version = "^0.19.0", markers = "platform_machine  == 'x86_64'"}
```

Comment:
we need to upgrade tensorflow-addons here to 0.20.0
https://github.com/tensorflow/addons#python-op-compatibility-matrix